src/components/List.js

```diff
@@ -79,6 +80,21 @@ const List = ({ token }) => {
     return daysUntilNextPurchase > 0 ? daysUntilNextPurchase : 0;
   };
 
+  // Confirm that the user would like to delete this item
+  const confirmDelete = (item) => {
+    const confirmed = window.confirm(
```

Comment:
We decided to use a confirm() pop up to prompt the user on whether or not they really want to delete the item. This can be changed to a styled modal or other pop up, but we didn't want to add too much styling prematurely without consulting the rest of the group.

Reply:
Nice.